PhpcsChanged/Cli.php

```diff
@@ -355,13 +363,24 @@ function runGitWorkflowForFile(string $gitFile, array $options, ShellOperator $s
 				$cache->setCacheForFile($gitFile, 'new', $newFileHash, $phpcsStandard ?? '', $newFilePhpcsOutput);
 			}
 		}
-		$fileName = DiffLineMap::getFileNameFromDiff($unifiedDiff);
+
+		$fileName = $shell->getFileNameFromPath($gitFile);
 		$newFilePhpcsMessages = PhpcsMessages::fromPhpcsJson($newFilePhpcsOutput, $fileName);
-		$hasNewFilePhpcsIssues = !empty($newFilePhpcsMessages->getMessages());
+		$hasNewPhpcsMessages = !empty($newFilePhpcsMessages->getMessages());
```

Comment:
I like the new variable name way better than the old one, makes the code more readable.